packages/react-components/src/organisms/RichText.tsx

```diff
@@ -59,13 +59,22 @@ const components = {
     ),
   h2: ({ children, id }: HTMLAttributes<HTMLHeadingElement>) =>
     isAllowedChildren(children) ? (
-      <Headline3 id={id}>{children}</Headline3>
+      <Headline5 id={id}>{children}</Headline5>
     ) : (
       <ErrorCard
         title="Styling Error"
         description="Invalid h2 heading styling"
       />
     ),
+  h3: ({ children, id }: HTMLAttributes<HTMLHeadingElement>) =>
+    isAllowedChildren(children) ? (
+      <Paragraph id={id}>{children}</Paragraph>
```

Comment:
Can we extract a `Headline6` for this please. I know it currently has no extra same styling, but that's a design coincidence, and it should still be `<h6>` and the `id` that is needed for headlines need not leak into `Paragraph`